src/ansiblelint/prerun.py

```diff
@@ -106,13 +106,13 @@ def install_collection(collection: str) -> None:
         "ansible-galaxy",
         "collection",
         "install",
-        "-p",
-        f"{options.cache_dir}/collections",
         "-v",
-        f"{collection}",
     ]
+    if destination:
+        cmd.extend(["-p", destination])
+    cmd.append(f"{collection}")
 
-    _logger.info("Running %s", " ".join(cmd))
+    _logger.warning("Running %s", " ".join(cmd))
```

Comment:
This does not sound like a warning since it is a normal operation (there is nothing users can do to get rid of that warning). I would leave it silenced by default or make the default log level INFO.